engine/src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -58,6 +57,28 @@ object LocalBackend {
     def script = jobDescriptor.callRootPath.resolve("script")
     def returnCode = jobDescriptor.callRootPath.resolve("rc")
   }
+}
+
+
+/**
+ * Handles both local Docker runs as well as local direct command line executions.
+ */
+case class LocalBackend(config: Config, actorSystem: ActorSystem) extends Backend with SharedFileSystem {
```

Comment:
I think Local backend should take care for it's own way of defining configuration and we should not standarize the configuration for all backends. I.e.: Getting configuration from companion object.